### DISCUSSION TITLE
Update from 6.9.x to 10.0.x - non-existent service "security.authentication.trust_resolver"

### DISCUSSION THREAD

**Discussion by mnadeaufbo** [pimcore/pimcore#9636] [Q&A / Support] [answered]
After a composer update, the clearcache script run but this is what it gives me : 

Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException^ {#9741
  -id: "security.authentication.trust_resolver"
  -sourceId: "scheb_two_factor.security.authentication.trust_resolver"
  -alternatives: []
  #message: "The service "scheb_two_factor.security.authentication.trust_resolver" has a dependency on a non-existent service "security.authentication.trust_resolver"."
  #code: 0
  #file: "./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/DecoratorServicePass.php"
  #line: 97
  trace: {
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/DecoratorServicePass.php:97 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/Compiler.php:91 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:749 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:545 { …}
    ./vendor/pimcore/pimcore/lib/Kernel.php:223 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:786 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:125 { …}
    ./vendor/pimcore/pimcore/lib/Kernel.php:202 { …}
    ./vendor/pimcore/pimcore/lib/Bootstrap.php:256 { …}
    ./vendor/pimcore/pimcore/lib/Bootstrap.php:60 { …}
    ./bin/console:24 {
      › /** @var \Pimcore\Kernel $kernel */
      › $kernel = \Pimcore\Bootstrap::startupCli();
      › $application = new \Pimcore\Console\Application($kernel);
    }
  }
}

What does this means, did I forgot a config parameter ?

**Comment by ShaileshT-dot** [comment, marked as answer]
Please make sure that the var/config folder is rightly updated. You may take reference of pimcore/skeleton in Github.

**Reply by mnadeaufbo** [reply]
security.yaml was the one missing some parameters.
Made a copy paste of the pimcore/skeleton one and it's now solved.

**Comment by triongroup** [comment]
Hi! I got the same error. @mnadeaufbo could you explain what exactly you changed in config/packages/security.yaml? I've copied this file from skeleton but its still not working.
